shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/resultset/ShardingSphereResultSet.java

```diff
@@ -17,6 +17,10 @@
 
 package org.apache.shardingsphere.driver.jdbc.core.resultset;
 
+import java.sql.SQLFeatureNotSupportedException;
```

Comment:
Add test case in ShardingSphereResultSetTest

Reply:
done